lib/hammer_cli_foreman_remote_execution/interface_extensions.rb

```diff
@@ -0,0 +1,13 @@
+require 'hammer_cli_foreman/interface'
+
+module HammerCLIForemanRemoteExecution
+  class InterfaceExtensions < ::HammerCLI::CommandExtensions
+    output do |definition|
+      definition.append do
+        field :execution, _('Execution')
```

Comment:
Could you append `, Fields::Boolean` to this line so the value gets nicely formatted?

Reply:
yup!